### PR TITLE
add zh trans for docs/reference/glossary/addons.md

### PR DESCRIPTION
**Pull request by ydcool**

### CODE REVIEW

File: content/zh/docs/reference/glossary/addons.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Resources that extend the functionality of Kubernetes.
+-->
+
+
+<!--more-->
+
+<!-- 
+[Installing addons](/docs/concepts/cluster-administration/addons/) explains more about using add-ons with your cluster, and lists some popular add-ons. 
+-->
+[安装扩展](/docs/concepts/cluster-administration/addons/) 阐释了更多关于如何在集群内使用扩展组件，并列出了一些流行的扩展组件。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/addons.md:37]: 如果 addon/add-on 翻译为“附加组件”，则至少在本页内要保持一致，不能再用“扩展”，“扩展组件”。

Comment (ydcool) [reply content/zh/docs/reference/glossary/addons.md]: updated

File: content/zh/docs/reference/glossary/addons.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  Resources that extend the functionality of Kubernetes.
+-->
+
+
+<!--more-->
+
+<!-- 
+[Installing addons](/docs/concepts/cluster-administration/addons/) explains more about using add-ons with your cluster, and lists some popular add-ons. 
+-->
+[安装附件组件](/docs/concepts/cluster-administration/addons/) 阐释了更多关于如何在集群内使用附加组件，并列出了一些流行的附加组件。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/addons.md:37]:
```suggestion
[安装附加组件](/docs/concepts/cluster-administration/addons/) 阐释了更多关于如何在集群内使用附加组件，并列出了一些流行的附加组件。
```

**Comment by tengqm** [comment]
please squash your commits for this to be approved, thanks.

**Comment by ydcool** [comment]
@tengqm Force pushed, thank you.

**Pull request merged**